index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Playfair+Display&family=Poppins&display=swap" rel="stylesheet">
+</head>
+<body>
+    <section class="flexcontainer">
+        <nav class="toolbar">
+            <img src="img/image_geometry_1.svg" alt="">
+            <img src="img/ic_menu.svg" alt="">
+        </nav>
```

Comment:
- [ ] Kindly make sure this section has a header tag container, currently, you have a nav tag container.
- [ ] Kindly make sure only the hamburger button is on the right and has a nav tag, also check on the hamburger menu positioning there needs to be some margin on the right to match the design given.
<img width="218" alt="image" src="https://user-images.githubusercontent.com/61408860/171015403-24e5e230-1800-4221-8531-7ce4d2c73aeb.png">

Reply:
Will do